govc/fields/ls.go

```diff
@@ -64,10 +68,17 @@ func (cmd *ls) Run(ctx context.Context, f *flag.FlagSet) error {
 		return err
 	}
 
-	tw := tabwriter.NewWriter(os.Stdout, 2, 0, 2, ' ', 0)
+	tw := tabwriter.NewWriter(os.Stdout, 3, 0, 2, ' ', 0)
 
 	for _, def := range field {
-		fmt.Fprintf(tw, "%d\t%s\n", def.Key, def.Name)
+		if cmd.mot == "" {
```

Comment:
Looks like output is the same in both conditions, can you collapse this?  As in:
```go
if cmd.mot == "" || cmd.mot == def.ManagedObjectType {
    fmt.Printf(...)
}
```

Reply:
ya this is cleaner, not sure what i was thinking.